internal/vault/secrets.go

```diff
@@ -3,36 +3,97 @@ package vault
 import (
 	"errors"
 	"net/url"
+	"strings"
 )
 
 var (
 	ErrInvalidSecret = errors.New("invalid vault secret")
 )
 
 const (
+	// PKI secret
 	queryKeyAltNames  = "altNames"
 	queryKeyIPSANs    = "ipSans"
 	queryKeyOtherSANs = "otherSans"
 	queryKeyTTL       = "ttl"
+
+	// Static secret
+	queryKeyCertKey       = "tlsCertKey"
+	queryKeyPrivateKeyKey = "tlsPrivateKeyKey"
```

Comment:
This reads a little weird but is parroting [the K8s Secret API](https://github.com/kubernetes/api/blob/d10499434b5204901e2681f883e988f35b63e2ba/core/v1/types.go#L6224-L6225). Not really glued to it though if someone has a different idea for naming

Reply:
@andrewstucki any ideas here?

Reply:
yeah, sorry, issued my review but somehow the suggestion I made got dropped. Just so we don't talk about using a key to retrieve a key to get the proper keys within a key-value entry, maybe something like:

`queryPrivateKeyField = "tlsPrivateKeyField"` (i.e. use `field` instead of `key` in this context and just simplify `queryKey*` to `query*`)?

Reply:
I like that, will make that change! Thx